examples/flutter_gallery/test_driver/transitions_perf_test.dart

```diff
@@ -136,14 +134,16 @@ void main() {
       // Save the duration (in microseconds) of the first timeline Frame event
       // that follows a 'Start Transition' event. The Gallery app adds a
       // 'Start Transition' event when a demo is launched (see GalleryItem).
+      await _fs.directory(testOutputsDirectory).create(recursive: true);
```

Comment:
Those actually already do that. Moved the create call into the write method above to keep it all symmetric.